templates/CRM/Event/Page/EventInfo.tpl

```diff
@@ -115,7 +115,7 @@
   {/if}
   <div class="clear"></div>
   <div class="crm-section event_date_time-section">
-      <div class="label"><label>{ts}When{/ts}</label></div>
```

Comment:
Me too, though it truly is becaiuse I don't know.  Here is a solid reference that suggests a different best practice, I think: https://www.w3.org/WAI/tutorials/forms/labels/

Reply:
@seamuslee001 @JoeMurray while was using WAVE accessibility evaluation tool, it throws a warning at these labels saying these are orphan labels and must be associated with form control (i.e. input field). Since there isn't any, the tool suggests to remove it, the reason why I removed the label tag. 

See :
![screen shot 2018-04-05 at 9 05 06 am](https://user-images.githubusercontent.com/3735621/38346095-9808e920-38b0-11e8-80a4-ef779fa02650.png)